src/test/integration/tests/Delegate_Deposit_Queue_Complete.t.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import "src/test/integration/IntegrationChecks.t.sol";
+import "src/test/integration/User.t.sol";
+
+contract Integration_Delegate_Deposit_QueueWithdrawal_Complete is IntegrationCheckUtils {
+    
+    function testFuzz_delegate_deposit_queueWithdrawal_completeAsShares(uint24 _random) public {
+        // Configure the random parameters for the test
+        _configRand({
+            _randomSeed: _random,
+            _assetTypes: HOLDS_LST | HOLDS_ETH | HOLDS_ALL,
+            _userTypes: DEFAULT | ALT_METHODS
+        });
+
+        // Create a staker and an operator with a nonzero balance and corresponding strategies
+        (User staker, IStrategy[] memory strategies, uint[] memory tokenBalances) = _newRandomStaker();
+        (User operator, ,) = _newRandomOperator();
+
+        // 1. Delegate to operator
+        staker.delegateTo(operator);
+        check_Delegation_State(staker, operator, strategies, new uint256[](strategies.length)); // Initial shares are zero
+
+        // 2. Deposit into strategy
+        staker.depositIntoEigenlayer(strategies, tokenBalances);
+        uint[] memory shares = _calculateExpectedShares(strategies, tokenBalances);
+
+        // Check that the deposit increased operator shares the staker is delegated to
+        check_Delegation_State(staker, operator, strategies, shares);
```

Comment:
Looks like everything passes except the two tests in this file. Reason is that this check helper function specifically assumes that the Staker has deposited and then delegated. As a result, the test checks that the Staker's shares _have not_ changed when they should have. Would recommend creating a new helper method that isolates this specific test case where the Staker's shares do increase after a deposit (because of previously delegating).